### PR TITLE
Fix syntax errors in README.md

### PR DESCRIPTION
**Pull request by karta-bello**
image

### CODE REVIEW

**Comment by miquelbeltran** [comment]
Although I appreciate the fixes, some of them are changes from British English to American English, and I'd like to respect the original Readme with British English. I will add some comments in the review.

File: README.md [hunk 1]
```diff
@@ -46,14 +46,14 @@ To build and run the mobile apps you’ll need to install [Flutter](https://flut
 $ flutter doctor
 ```
 
-The app is optimised for Android and iOS phones in portrait mode.
+The app is optimized for Android and iOS phones in portrait mode.
```
Comment (miquelbeltran) [inline README.md]: Let's keep the 🇬🇧 spelling <3

File: README.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ $ firebase deploy --only functions
 
 ### B5. Final steps
 
-Run the flutter app using your favourite IDE (e.g. Visual Studio Code / Android Studio). Next you’ll need to run the app. 
+Run the flutter app using your favorite IDE (e.g. Visual Studio Code / Android Studio). Next, you’ll need to run the app.
```
Comment (miquelbeltran) [inline README.md]: Let's keep the 🇬🇧 spelling

File: README.md [hunk 3]
```diff
@@ -174,6 +174,6 @@ When building for release the app will automatically use the production configur
 # About
 
 The concept for Timy was created and developed by [kaalita](https://github.com/kaalita) and [philippmoeser](https://github.com/philippmoeser).
-The the initial version is a MVP messaging app focusing on organising events among groups.
+The initial version is a MVP messaging app focusing on organizing events among groups.
```
Comment (miquelbeltran) [inline README.md]: Let's keep the 🇬🇧 spelling

Comment (karta-bello) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (karta-bello) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (karta-bello) [reply README.md]: Fixed

**Pull request merged**